src/hyperion/experiment_plans/robot_load_then_centre_plan.py

```diff
@@ -113,10 +113,34 @@ def wait_for_smargon_not_disabled(smargon: Smargon, timeout=60):
     )
 
 
+def prepare_for_robot_load(composite: RobotLoadThenCentreComposite):
+    yield from bps.abs_set(
+        composite.aperture_scatterguard,
+        AperturePositions.ROBOT_LOAD,
+        group="prepare_robot_load",
+    )
+
+    yield from bps.mv(composite.smargon.stub_offsets, StubPosition.RESET_TO_ROBOT_LOAD)
+
+    # fmt: off
```

Comment:
What is fmt here?

Reply:
Turns `black` off. If you let `black` reformat the line it becomes less readable (I think) as it's not obvious which value corresponds to which axis.